oprheum-game/SnowballNode.swift

```diff
@@ -22,6 +22,7 @@ class SnowballNode: SKNode {
     }
 
     func setSnowballMass(mass: CGFloat) {
+        if mass>1.6 {let mass = 1.8}
```

Comment:
Pretty sure this doesn't work.
![simulator screen shot oct 4 2016 6 01 07 pm](https://cloud.githubusercontent.com/assets/930039/19095661/9d40715a-8a5c-11e6-895e-902ae00f95fd.png)